app/controllers/admin/users_controller.rb

```diff
@@ -1,2 +1,29 @@
 class Admin::UsersController < Admin::BaseController
+  expose :user
+  expose :interests, from: :user
+  expose_decorated(:users, -> { User.all })
+
+  def create
+    user.password = 'secret'
```

Comment:
Great! Can you see now, what missing in this PR is? :)
Please, always add a PR's description to describe what exactly has been done. If the code and commits are not a self explonatory it's harde to get it from context without the informations and test scenario.